src/composite.h

```diff
@@ -0,0 +1,17 @@
+#ifndef SRC_COMPOSITE_H_
+#define SRC_COMPOSITE_H_
+
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+/*
+  Composite images `src` and `dst`.
+*/
+int Composite(VipsObject *context, VipsImage *src, VipsImage *dst, VipsImage **out);
```

Comment:
Haha, good one :stuck_out_tongue_winking_eye: On a more serious note: I do it because I like to do three-way merges on my laptop :+1: